.github/workflows/conda-cpp-post-build-checks.yaml

```
@@ -0,0 +1,81 @@
+on:
+  workflow_call:
+    inputs:
+      build_type:
+        required: true
+        type: string
+      branch:
+        type: string
+      date:
+        type: string
+      sha:
+        type: string
+      repo:
+        type: string
+      enable_check_symbols:
+        default: false
+        type: boolean
+        required: false
+      symbol_exclusions:
+        type: string
+        default: "void (cub::|thrust::)"
+
+defaults:
+  run:
+    shell: bash
+
+permissions:
+  actions: read
+  checks: none
+  contents: read
+  deployments: none
+  discussions: none
+  id-token: write
+  issues: none
+  packages: read
+  pages: none
+  pull-requests: read
+  repository-projects: none
+  security-events: none
+  statuses: none
+
+jobs:
+  check-symbols:
+    if: ${{ inputs.enable_check_symbols }}
+    runs-on: linux-amd64-cpu4
+    container:
+      image: rapidsai/ci-wheel:cuda12.0.1-centos7-py3.10
```

```suggestion
      image: rapidsai/ci-wheel:latest
```

Does the version of this container matter?

If not, can we switch it to `latest`?

The versions used for our `latest` tags are specified in the `rapidsai/ci-imgs` repository here: https://github.com/rapidsai/ci-imgs/blob/main/latest.yaml

---

The version doesn't matter. So moving to use `latest`